docs/en/observability/monitor-k8s/monitor-k8s.asciidoc

```diff
@@ -35,9 +35,11 @@ your application, including the orchestration software itself:
 
 //TODO: Add APM goals here
 
-This guide describes how to deploy Elastic monitoring agents as DaemonSets.
-For other deployment options, see {eck-ref}/index.html[{ecloud} on Kubernetes (ECK)]
-and https://github.com/elastic/helm-charts/blob/master/README.md[Helm charts].
+This guide describes how to deploy Elastic monitoring agents as DaemonSets using
+`kubectl` and the manifest files from the Beats GitHub repository. For other
+deployment options, see the Kubernetes operator and custom resource definitions
+from {eck-ref}/index.html[{ecloud} on Kubernetes (ECK)] or the
```

Comment:
@masci I assume we want to keep these as active links, no?

Reply:
Yes, sorry that was some bad copypasta